metricbeat/docs/modules/gcp.asciidoc

```diff
@@ -329,6 +329,13 @@ metricbeat.modules:
   dataset_id: "dataset id"
   table_pattern: "table pattern"
   cost_type: "regular"
+  metricsets:
```

Comment:
Should this be under it's own `- module: gcp` section?  As it stands, it looks like it's part of the previous `- module: gcp` section that has the `billing` metricset in it.

Reply:
That's absolutely true. I guess the way it was overrides the billing metricset. I'm surprised no tests failed.

Reply:
Scratch my previous comment. This file is auto-generated. So There may be something wrong somewhere, but I can't just change this content here.